claim/README.md

```diff
@@ -194,6 +194,20 @@ might be having with the ACLK or claiming process.
 
 Use these keys and the information below to troubleshoot the ACLK.
 
+#### bash: netdata-claim.sh: command not found
+
+If you run the claiming script and see a `command not found` error, the claiming script is not in your system's `$PATH`.
+Netdata expects this file at `/usr/sbin/`, but some systems may have installed the script in `/opt/netdata/usr/sbin/` or
+`/opt/netdata/bin/`.
+
+Check those locations for the existence of the `netdata-claim.sh` file. When you find it, run the following commands to
+claim your node.
+
+```bash
+sudo chmod +x netdata-claim.sh
```

Comment:
Marking it executable should never be needed, same as the other comment if this _is_ necessary, it’s a bug and should be handled as such.

Reply:
Good call, thanks—I'll probably just remove this block entirely if we go the route I described in your other comment.